libraries/botbuilder-dialogs-adaptive-testing/src/httpRequestMocks/httpRequestSequenceMock.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @module botbuilder-dialogs-adaptive-testing
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { HttpMethod } from 'botbuilder-dialogs-adaptive';
+import * as nock from 'nock';
+import * as parse from 'url-parse';
+import { HttpRequestMock } from './httpRequestMock';
+import { HttpResponseMock } from './httpResponseMock';
+import { SequenceResponseManager } from './sequenceResponseManager';
+
+export enum BodyMatchType {
+    Exact = 'Exact',
+    Partial = 'Partial',
+}
+
+export interface HttpRequestSequenceMockConfiguration {
+    method?: string | HttpMethod;
+    url?: string;
+    matchType?: string | BodyMatchType;
+    body?: string;
+    responses?: HttpResponseMock[];
+}
+
+export class HttpRequestSequenceMock extends HttpRequestMock implements HttpRequestSequenceMockConfiguration {
+    public static $kind = 'Microsoft.Test.HttpRequestSequenceMock';
+
+    public method: HttpMethod;
+    public url: string;
+    public matchType: BodyMatchType = BodyMatchType.Partial;
```

Comment:
I think you can omit the explicit `: BodyMatchType` piece as it should be inferred from the default value.